app/jobs/course/assessment/submission/auto_grading_job.rb

```diff
@@ -11,7 +11,7 @@ class Course::Assessment::Submission::AutoGradingJob < ApplicationJob
   #   results into.
   # @param [Boolean] only_ungraded Whether grading should be done ONLY for
   #   ungraded_answers, or for all answers regardless of workflow state
-  def perform_tracked(submission, only_ungraded: false)
+  def perform_tracked(submission, only_ungraded = false) # rubocop:disable Style/OptionalBooleanParameter
```

Comment:
Any reason for doing `=` instead of `:`? Just curious.

Reply:
In Rails API, the perform_later method takes in *args as the only argument. As such, if we pass in a keyword argument here (https://github.com/Coursemology/coursemology2/pull/4165/files#diff-11bda8445480301b60c1f569535d4c5f6c3aa44819fa7b2cecd08fbdd4f73591R180), it cant be properly passed to the perform_tracked method based on the new ruby 2.7 kwargs update.

Reply:
So if we want to use 
`def perform_tracked(submission, only_ungraded: false)`,
we need to pass
AutoGradingJob.perform_later([self, only_ungraded: only_ungraded])

Reply:
I see, so there's some difference between an optional param vs a keyword param with a default value, and it seems like Rubocop prefers the latter?

Reply:
for boolean only, yes rubocop prefers the later